dbt/adapters/athena/impl.py

```diff
@@ -385,28 +386,38 @@ def _get_one_catalog(
     ) -> agate.Table:
         data_catalog = self._get_data_catalog(information_schema.path.database)
         catalog_id = get_catalog_id(data_catalog)
```

Comment:
it seems `catalog_id` has no sense for non GLUE catalogues. Let's put this call after 
```
if data_catalog["Type"] == "GLUE":
```